app/assets/stylesheets/pages/_docs.scss

```diff
@@ -200,8 +200,8 @@ $color-green: rgb(3, 165, 98);
 }
 
 a.Docs__example-repo {
-  align-items: top;
   border: 1px solid #ccc;
+  color: map-get($color-aliases, "text-base");
```

Comment:
Ensure purple link color won't mess with the example repo color, which sits within `.TextContent`.

It would be good to refactor non-markdown links in a future PR.